### DISCUSSION TITLE
wrong dialog on request for elevated privileges

### DISCUSSION THREAD

**Discussion by AonH** [rustdesk/rustdesk#4914] [General]
### Bug Description

The dialog displayed when trying to request elevated privileges "The remote user must still click "yes" in RustDesk's UAC window" is incorrect.
![client](https://github.com/rustdesk/rustdesk/assets/86567756/7422f067-5408-4564-a3f5-78cad31f05a5)

The UAC prompt for the user appears on the target, but without a password, clicking on the "YES" button goes around in circles. Here, the dialog should be changed to "The remote user must still click "**NO**" in RustDesk's UAC window" so that the client restarts with the transmitted credentials of the remote computer. After that, UAC input from the remote client is possible without any problems.
![host](https://github.com/rustdesk/rustdesk/assets/86567756/772160e9-a4e7-4861-afa8-63b7e3857576)

### How to Reproduce

Request elevated privileges and submit Credentials

### Expected Behavior

change the Dialog from "yes" to "no"

### Operating system(s) on local side and remote side

WIN -> WIN

### RustDesk Version(s) on local side and remote side

1.2.0 -> 1.2.0

### Screenshots

![client](https://github.com/rustdesk/rustdesk/assets/86567756/0bf59733-3119-469e-af84-acef63d68ada)
![host](https://github.com/rustdesk/rustdesk/assets/86567756/9c8442c7-5dee-43ee-af6c-c12072db7ae0)

**Comment by rustdesk** [comment]
Frankly, no idea what you are saying.
